src/recipe/recipe.cache.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@nestjs/common';
+import { RedisCacheService } from '../cache/redisCache.service';
+import { Recipe } from '@prisma/client';
+
+function recipeCacheKey(recipeId: number): string {
+  return `cache:recipe:${recipeId}`;
+}
```

Comment:
hmm, why is this function outside of class? You can define it as a private method in `RecipeCacheService` class - it won't be publicly available (and I don't believe it there should be a need for it being public)

Reply:
wanted to call `recipeCacheKey(...)` rather than `this.recipeCacheKey(...)`